CorePayments/src/main/java/com/paypal/android/corepayments/analytics/AnalyticsService.kt

```diff
@@ -43,7 +43,7 @@ class AnalyticsService internal constructor(
     fun sendAnalyticsEvent(name: String, orderId: String?) {
         // TODO: send analytics event using WorkManager (supports coroutines) to avoid lint error
         // thrown because we don't use the Deferred result
-        scope.async {
+        scope.launch {
```

Comment:
`launch()` is more appropriate here–`async()` typically has a balancing call to `await()`. In the long-term, once we migrate to WorkManager this will no longer be required.